st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -61,19 +63,22 @@ def __init__(self):
         self.get_one_db_method = self._get_by_name
 
     @jsexpose(arg_types=[str, str, str, bool])
-    def get_one(self, name, scope=SYSTEM_SCOPE, user=None, decrypt=False):
+    def get_one(self, name, scope=None, user=None, decrypt=False):
         """
             List key by name.
 
             Handle:
                 GET /keys/key1
         """
-        self._validate_scope(scope=scope)
+        if not scope:
+            scope = SYSTEM_SCOPE
```

Comment:
Assumption: If `NoneType` is assigned to method call it still gets replaced with `SYSTEM_SCOPE`?